golden_test/testing/font_loader.dart

```diff
@@ -35,10 +36,29 @@ Future<Map<String, List<Future<ByteData>>>> loadFontsFromManifest() async {
   return fontFamilyToData;
 }
 
+Map<String, List<Future<ByteData>>> loadFontsFromTestingDir() {
```

Comment:
Why do all the roboto font files have to be included now?

Reply:
We were not using the Roboto font on the home page, but it is on the demo page, so I never noticed that we needed to load it as well.

In the previous code we were only loading the google fonts that were added and the icons from the `pubspec.yaml`. As the Roboto font is added by default in Flutter and not added in the `pubspec.yaml` or in the `google_fonts` folder, we need to explicitly load it here in the tests.